src/style.css

```diff
@@ -0,0 +1,45 @@
+
+body {
+  margin: 0;
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
What about the last elements in a row? They should not have margin-right
![image](https://github.com/mate-academy/layout_stars/assets/96409563/c7a44a1b-ccd2-4955-87c0-af40108a8ecc)